### PR TITLE
[TDT-386] Added in readme repos to be consistently updated with the release

### PR DESCRIPTION
**Pull request by anabrs**
[TDT-386](https://crowdflower.atlassian.net/browse/TDT-386)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,6 +174,24 @@ summary.get({ service: 'database' })
 # => { 0.5 => 0.1233122, 0.9 => 3.4323, 0.99 => 5.3428231 }
 ```
 
+## Releases
+
+When a new version of this gem is released, the tags within the gemfile in the following repositories must be updated.
+
+* [Akon client](https://github.com/CrowdFlower/akon_client)
```
Comment (mrgordon) [inline README.md]: I didn't see this dependency in Akon Client. Maybe we mean Akon?

Comment (anabrs) [reply README.md]: Ahhh you are right! Thank you for noticing! I'll change it in the others too.

**Comment by maj** [comment]
@anabrs Please assign some people to review your pull requests. Thanks!

File: README.md [hunk 2]
```diff
@@ -174,6 +174,24 @@ summary.get({ service: 'database' })
...
+* [Builder](https://github.com/CrowdFlower/builder)
```
Comment (maj) [inline README.md]: FYI same issues as https://github.com/CrowdFlower/feature_switch/pull/6

**Review by maj — CHANGES_REQUESTED** [review]
Looks good once the readme is updated

**Review by maj — APPROVED** [review]

**Review by mflores88 — APPROVED** [review]

**Pull request merged**